config/config_test.go

```diff
@@ -250,6 +251,12 @@ func TestDDAgentConfigWithNewOpts(t *testing.T) {
 }
 
 func TestDDAgentConfigBothVersions(t *testing.T) {
+	origcfg := config.Datadog
+	config.Datadog = config.NewConfig("datadog", "DD", strings.NewReplacer(".", "_"))
```

Comment:
since `config.NewConfig("datadog", "DD", strings.NewReplacer(".", "_"))` is often used should we have a helper for it ?

Reply:
It's just tests + a one-liner so I didn't feel too strongly.

Reply:
For the `origcfg` we could just store it as a global variable in the test file and have a `resetGlobalConfig` helper

Reply:
Makes sense - I'll do that